src/tests/unit/services/email/email-provider-factory.test.ts

```
@@ -0,0 +1,148 @@
+
+    it('should fall back to NODE_ENV when config.environment is not set', () => {
+      mockConfig.environment = ''
+      process.env.NODE_ENV = 'test'
+      
+      const provider = EmailProviderFactory.create()
+      expect(provider).toBeInstanceOf(MockEmailService)
+    })
+
+    it('should default to development when no environment is set', () => {
+      mockConfig.environment = ''
+      delete process.env.NODE_ENV
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)